packages/quantic/force-app/main/default/lwc/quanticDocumentSuggestion/quanticDocumentSuggestion.js

```diff
@@ -31,12 +31,16 @@ export default class QuanticDocumentSuggestion extends LightningElement {
    */
   @api engineId;
   /**
-   * Whether or not we want to display the quick view for the document suggestions.
-   * @api
-   * @type {boolean}
-   * @defaultValue `false`
+   * @deprecated
```

Comment:
Even if it's marked as deprecated, it still has to work until we remove it. Is that the case?

Reply:
Even though the `show-quickview` attribute is not considered in the code anymore, I think it is covered.

Suppose a customer wants the quickview to be shown (before the change). He would declare the component like this.

```html
<c-quantic-document-suggestion show-quickview />
```
After the upgrade, quickview will be shown because `hide-quickview` is not specified.

Reply:
that's how it's working right now:
<img width="311" alt="how_it's_working" src="https://user-images.githubusercontent.com/86681870/159020593-7b20b5f2-32e6-49d6-a898-ef4bdb7a72b2.png">

There is one case where the customer could suffer from a behaviour change and it's the case you mentioned @lbergeron: 
> Unfortunately, this is introducing a behavior change for an implementation where the customer does not want the quick view to be shown. Since the `show-quickview` attribute would not be specified, the quickview will appear after the upgrade.

I don't see a solution to prevent this🧐 the customer will need to set the hide-quickview attribute to get  back to his usual behaviour.
Can we afford to do this change and rely on documentation to make this easy for customers?